pypeit/reduce.py

```diff
@@ -354,9 +356,17 @@ def run(self, basename=None, ra=None, dec=None, obstime=None,
                               show=self.reduce_show & (not self.std_redux),
                               manual_extract_dict=manual_extract_dict)
 
+        # Check if the user wants to overwrite the skymask with a pre-defined sky regions file
+        skymask_init, usersky = self.load_skyregions(skymask_init)
+
+        # Define the trim regions for different pypelines
+        if self.spectrograph.pypeline == "IFU":
+            trim_edg = (0, 0)
+        else:
+            trim_edg = (3, 3)
+
         # Global sky subtract
-        self.initial_sky = \
-            self.global_skysub(skymask=skymask_init).copy()
+        self.initial_sky = self.global_skysub(skymask=skymask_init, trim_edg=trim_edg).copy()
```

Comment:
Remove the trim_edge, this can be handled via parsets.

Reply:
Done!